public/stylesheets/style.css

```diff
@@ -7,32 +7,43 @@
         color: #00B7FF;
     }
 
-    #header.parent{
-      position: relative;
+    #header.parent {
+        position: relative;
     }
 
     #header.child {
-      position: absolute;
-      top: 50%;
-      transform: translateY(-50%);
+        position: absolute;
+        top: 50%;
+        transform: translateY(-50%);
     }
-
-    .search-form {
+    /*.search-form {
         display: -moz-flex;
         display: -webkit-flex;
         display: -ms-flex;
         display: flex;
         position: relative;
         margin: auto;
         width: 25em;
+    }*/
+
+    #searchBox.index{
+        width: 30em;
+        margin: auto;
+    }
+
+    #searchInput {
+        width: 30em;
     }
 
     @media screen and (max-width: 480px) {
-        .search-form {
-            width: 20em;
+        #searchBox.index{
```

Comment:
add a space between the bracket and the text. Do likewise for the change on line 29.